.github/workflows/notify-teams-channel.yml

```diff
@@ -1,4 +1,4 @@
-name: Post PRs and issues to Teams
+name: notify-teams-channel
```

Comment:
Updated the name that is displayed in the GitHub Checks area to match the naming convention we have for the other checks.